app/policies/bookmark_policy.rb

```diff
@@ -1,18 +1,18 @@
 class BookmarkPolicy < ApplicationPolicy
   def create?
-    index?
+    @record.user == @user
```

Comment:
@parndt does the record already have a user at this stage?

Reply:
yep!

When running:
```
bundle exec rspec spec/controllers/bookmarks_controller_spec.rb:80
```

which is this spec:
```ruby
it "allow creating a bookmark" do
  sign_in user

  expect(subject).to have_http_status(201)
end
```

It hits this:

```ruby
From: marine-defrag-server/app/policies/bookmark_policy.rb:3 BookmarkPolicy#create?:

    2: def create?
 => 3:   binding.pry
    4:   @record.user == @user
    5: end

[1] pry(#<BookmarkPolicy>)> @record.user
=> #<User id: 1, email: "blair.hessel@schmidt.net", created_at: "2021-11-15 21:49:24.561012000 +0100", updated_at: "2021-11-15 21:49:24.561012000 +0100", name: "Andria Muller", provider: "email", uid: "blair.hessel@schmidt.net", updated_by_id: nil, allow_password_change: true, created_by_id: nil>
[2] pry(#<BookmarkPolicy>)> @user
=> #<User id: 1, email: "blair.hessel@schmidt.net", created_at: "2021-11-15 21:49:24.561012000 +0100", updated_at: "2021-11-15 21:49:24.561012000 +0100", name: "Andria Muller", provider: "email", uid: "blair.hessel@schmidt.net", updated_by_id: nil, allow_password_change: true, created_by_id: nil>
```

And returns this with `JSON.parse(subject.body)`:

```json
{"data"=>
  {"id"=>"4",
   "type"=>"bookmarks",
   "attributes"=>{"created_at"=>"2021-11-15T21:48:50+01:00", "updated_at"=>"2021-11-15T21:48:50+01:00", "updated_by_id"=>nil, "user_id"=>7, "title"=>"test 1", "view"=>{"lorem"=>"ipsum"}}}}
```